sdk/core/src/CommonNetworkMember/NetworkMemberWithCognito.ts

```diff
@@ -24,17 +24,20 @@ import { normalizeShortPassword } from '../shared/normalizeShortPassword'
 
 type UserDataWithCognito = ConstructorUserData & {
   cognitoUserTokens: CognitoUserTokens
+  createdDate: string
```

Comment:
we dont need to add it as a field or keep this data . we could just use  a call to service .

Reply:
Idea was to have it as a method on sdk